tools/scripts/ffi.toml

```diff
@@ -50,6 +52,26 @@ cd ffi/cpp/docs;
 exec make html
 '''
 
+[tasks.gen-c-diplomat]
```

Comment:
suggestion: test-ffi should require a task that:

 - depends on the three gen tasks
 - runs `git diff -- ffi/capi` and ensures there are no changes. it will fail locally if you have uncommitted changes but that's fine